ownCloud/Resources/Base.lproj/LaunchScreen.storyboard

```diff
@@ -13,7 +17,18 @@
                     <view key="view" contentMode="scaleToFill" id="Ze5-6b-2t3">
                         <rect key="frame" x="0.0" y="0.0" width="375" height="667"/>
                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
-                        <color key="backgroundColor" red="1" green="1" blue="1" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
+                        <subviews>
+                            <imageView clearsContextBeforeDrawing="NO" userInteractionEnabled="NO" contentMode="scaleAspectFit" horizontalHuggingPriority="251" verticalHuggingPriority="251" image="SplashImage.png" translatesAutoresizingMaskIntoConstraints="NO" id="WtJ-CJ-2he">
+                                <rect key="frame" x="0.0" y="20" width="375" height="647"/>
```

Comment:
I would scale the image canvas a little bit down, with a padding to the left and right side. Looks like a little bit oversized.

Reply:
How much do you want me to scale it? 30 points each side should be enough?

Reply:
I think you should be include a scaling factor. If you use a static constant, this does not make sense for every device size. On a iPad screen it needs much more spacing. Please can you play with a scaling factor and post a screenshot on a iPad 12.9 and iPhone SE?

Reply:
Take a look at the latest code, I've updated the constraints for every size class and I think looks better now.

Reply:
Thank you! Looks nice!